### ISSUE TITLE
你好

### ISSUE THREAD

**Issue by agingdata** [MarkPrecursor/EEDS-keras#1]
Hello, MarkPrecursor
   I was fortunate to study your EEDS code, I follow the readme run on the warehouse program, but I found a few questions:
1.prepare_data.py when the implementation of my machine, reported the error "error: D: \ Build \ OpenCV \ opencv-3.2.0 \ modules \ imgproc \ src \ color.cpp: 9781: error: (-215 ) scn == 3 || scn == 4 in function cv :: cvtColor ", I only know that this is the error in the opencv package, but I can not solve, I do not know where the problem lies
2. appear in EED.py and EEDS.py modules
data, label = pd.read_training_data ("./ train.h5")
  val_data, val_label = pd.read_training_data ("./ val.h5"), which is the train.h5 and val.h5 pre-training model, but I did not see the storage of both models in prepare_data.py.
I studied the procedures related to the paper and the program for many days, would like to gain. So, trouble you in the free time to give me some guidance. Thank you very much, from Henan, an ordinary university research students.
